src/Model/Filter/Base.php

```diff
@@ -188,13 +190,34 @@ public function skip()
     public function value()
     {
         $value = $this->_config['defaultValue'];
-        if (isset($this->_args[$this->name()])) {
-            $passedValue = $this->_args[$this->name()];
-            if (!is_array($passedValue) ||
-                $this->getConfig('multiValue')
-            ) {
-                return $passedValue;
-            }
+
+        $passedValue = $this->passedValue();
+        if ($passedValue === null) {
+            return $value;
+        }
+
+        if (!is_array($passedValue) ||
+            $this->getConfig('multiValue')
+        ) {
+            return $passedValue;
+        }
+
+        return $value;
+    }
+
+    /**
+     * @return string|array|null
+     */
+    public function passedValue()
```

Comment:
Having a new public method doesn't seem necessary. Can't think of a case where just calling `passedValue()` would be required without the additional stuff done in `value()`.

Reply:
Oh, I wanted to make this protected actually. My bad.

Reply:
I find it cleaner to wrap this into such a method instead of having too much nesting/logic inside value() itself.